graphz/src/main/scala/coop/rchain/graphz/Graphz.scala

```diff
@@ -1,11 +1,13 @@
 package coop.rchain.graphz
 
+import java.io.FileOutputStream
+
+import scala.collection.mutable.ListBuffer
```

Comment:
What's the motivation behind using a mutable collection inside the `MonadState`?

Reply:
@lukasz-golebiewski The reason for a mutable `ListBuffer` is that we want to append and not prepend lines of strings. Of course, we could prepend and then reverse the whole list later. But this seems error-prone to me.

Reply:
Why not use `Vector`?

Reply:
@lukasz-golebiewski np